common/src/main/java/io/netty/util/internal/StringUtil.java

```diff
@@ -314,6 +322,67 @@ public static String simpleClassName(Class<?> clazz) {
         }
     }
 
+    /**
+     * Escapes the specified value, if necessary according to
+     * <a href="https://tools.ietf.org/html/rfc4180#section-2">RFC-4180</a>.
+     * <p>
+     * The {@link CharSequence} will be escaped is if contains a
```

Comment:
On second thought lets either remove this block (the RFC defines the escaping rules) or clarify a bit:

```
The special characters are:
<ul>
...
</ul>
```

The value may not be "escaped" just because these characters are found (if it is already properly quoted and escaped nothing will be done).